harvard-lts/jstor_aspace_oai_harvester
# LTSK8S-493

**Configurable console logging.**
* * *

**JIRA Ticket**: [(link)](https://at-harvard.atlassian.net/browse/LTSK8S-493)

# What does this Pull Request do?
This pr introduces configurable console-only logging, by setting .env to CONSOLE_LOGGING_ONLY=true|false

# How should this be tested?

- Check out this branch
- Set console logging to true
- Build/start container, look for logging in console, confirm no logs written to filesys
- - docker-compose -f docker-compose-local.yml up --build -d --force-recreate
- Stop container
- Set console logging to false
- Build/start container (see above), look for logging on filesys (there will also be console)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
